src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -147,6 +152,17 @@ def RunfilesEnvvar(module_space):
 
   return (None, None)
 
+def BuildUserCustomizeModule(import_paths):
+  return """\
+# AUTOGENERATED FILE. DO NOT EDIT.
```

Comment:
Nit: Can you mention this is produced by the python launcher stub script? So anyone digging around later is saved a step.